README.md

```diff
@@ -111,28 +111,39 @@ To build and run test just run:
 gradle build --console=rich
 ```
 
-## Releasing
-
-In order to release to Central Repository, simply cut a new release tag in the Github UI. There is a configured [Github
-Action](./.github/workflows/publish.yml) that will automatically publish the release to Central Repository. You can cut
-the release by going to the [create a new releases page](https://github.com/TBD54566975/web5-kt/releases/new). When
-creating a new tag, the name should be in the format `vX.Y.Z`. Please note that once a release is made, it is immutable
-and cannot be deleted.
-
-### Manual Release
-
-If you want to do a manual release, you have two options:
-
-1. Dispatch the [publish workflow](./.github/workflows/publish.yml) workflow from the Github UI. Go to the [publish
-   Actions](https://github.com/TBD54566975/web5-kt/actions) > "Run workflow".
-2. Setup your local environment to publish to Central Repository. This is more involved. You'll need to:
-1. Define all the environment variables described in the [publish workflow](./.github/workflows/publish.yml) file. You
-   can find the values in the [secrets and variable](https://github.com/TBD54566975/web5-kt/settings/secrets/actions)
-   page.
-2. Run the following command (you can change `samplebranch` to any branch name):
-   ```bash
-   gradle -Pversion=samplebranch-SNAPSHOT publishToSonatype closeAndReleaseSonatypeStagingRepository
-   ```
+## Release Guidelines
+
+### Pre-releases
+
+In Kotlin we use the SNAPSHOT convention to build and publish a pre-release package that can be consumed for preview/tests purposes.
+
+To kick that off:
+
+1. Open the [Publish workflow](https://github.com/TBD54566975/web5-kt/actions/workflows/publish.yml), press the **Run workflow button** selecting the branch you want to generate the snapshot from.
+
+2. In the version field, insert the current version, a short meaningful identifier and the `-SNAPSHOT` prefix, ie:
+
+   - 0.11.0.pr123-SNAPSHOT
+   - 0.11.0.shortsha-SNAPSHOT
+   - 0.11.0.fixsomething-SNAPSHOT
+
+3. Run workflow!
+
+**DON'T FORGET THE `-SNAPSHOT` SUFFIX**, otherwise it will generate publish a new official release to maven registry.
```

Comment:
Hmmm I wonder if maybe we should implement something to prevent accidents here. Like, maybe we have a dedicated workflow for pre-release, even though everything would be the same as a production release, at least the developer would be more intentional from the page load? Idk I still don't love that idea... what could we do? 🤔

Reply:
Maybe somehow `SNAPSHOT` would be the default and it would require developer intervention to override it?

Reply:
@KendallWeihe - This is a great point; @leordev @finn-tbd and I are going to handle this as part of #217 . There will then be 2 workflows: one for SNAPSHOTs and one for releases. Each workflow will perform precondition checks on the user input for version to make sure the `-SNAPSHOT` suffix is either present or not depending upon the workflow.